webviews/createPullRequestView/index.css

```diff
@@ -84,4 +84,8 @@ body {
 
 .selector-group .wrapper {
 	padding-right: 5px;
-}
\ No newline at end of file
+}
+.select-style {
```

Comment:
Can these styles be added to the `select` selector instead of making a new class?

https://github.com/yash621/vscode-pull-request-github/blob/9dabb657592114854a9402a21e0520e54cd98dd9/webviews/createPullRequestView/index.css#L12-L14